Source/Engine/Platform/Win32/Win32Network.cpp

```diff
@@ -321,6 +323,28 @@ bool Win32Network::IsWriteable(NetworkSocket& socket)
     return false;
 }
 
+bool Win32Network::CreateSocketGroup(uint32 capacity, NetworkSocketGroup& group)
+{
+    if (!(group.Data = (byte*)malloc(capacity * SOCKGROUP_ITEMSIZE)))
+    {
+        LOG(Error, "Unable to malloc NetworkSocketGroup::Data ! Size : {0}", capacity * SOCKGROUP_ITEMSIZE);
+        return true;
+    }
+    group.Capacity = capacity;
+    for(int i = 0; i < (int)group.Capacity; i++)
+        ((pollfd*)&group.Data[i * SOCKGROUP_ITEMSIZE])->fd = -1;
+    
+    return false;
+}
+
+bool Win32Network::DestroySocketGroup(NetworkSocketGroup& group)
+{
+    if (!group.Data)
+        return true;
+    free(group.Data);
```

Comment:
Use `Platform::Free()` to unify memory allocations in the engine (with profiling).